CHANGELOG.md

```diff
@@ -111,6 +111,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added: Proper combat requirements for the Amorbis fight.
 - Removed: Incorrect and improper connections to and from the Amorbis fight.
 
+### Metroid Prime
+
+#### Logic Database
+
+- Added: 48 videos to logic database, bringing the total available via the [Video Directory](https://randovania.github.io/Metroid%20Prime/) 216
```

Comment:
`[...] to 216.`

Reply:
You also added the changelog to 5.6.0 instead of the top version.

Reply:
Understood, will `git pull --rebase origin` be what I use to grab all recent changes to main while keeping my changes intact?

Reply:
I'm not sure if `origin` is your fork or upstream in your case, so I can't say.

You can press the Update Branch button in GitHub here and solve the conflicts here. Just remember to pull after you do that.